rules/drop-capability-netraw/raw.rego

```diff
@@ -0,0 +1,77 @@
+package armo_builtins
```

Comment:
to prevent typing mistakes (of the failed path), you can use something like:
```
path := ["spec", "jobTemplate", "spec", "template", "spec", "containers"]
containers := object.get(wl, path, [])
container := containers[i]
...
fix_path := concat(".", path) + sprintf("[%d]", [i])
```